content/en/user-guide/aws/lambda/index.md

```diff
@@ -165,20 +165,51 @@ The product of 10 and 10 is 100%
 [Lambda event source mappings](https://docs.aws.amazon.com/lambda/latest/dg/invocation-eventsourcemapping.html) allows you to connect Lambda functions to other AWS services.
 The following event sources are supported in LocalStack:
 
+- [Simple Queue Service (SQS)](https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html)
 - [DynamoDB](https://docs.aws.amazon.com/lambda/latest/dg/with-ddb.html)
 - [Kinesis](https://docs.aws.amazon.com/lambda/latest/dg/with-kinesis.html)
 - [Managed Streaming for Apache Kafka (MSK)](https://docs.aws.amazon.com/lambda/latest/dg/with-msk.html)
-- [Simple Queue Service (SQS)](https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html)
-
-The limitations compared to AWS include:
-- Lambda Success Destinations are not supported.
-- Only basic validations are performed upon creating and updating ESM.
-- Streaming Pollers for Kinesis and DynamoDB do not implement the following batching behavior configurations:
-  - `BisectBatchOnFunctionError`
-  - `MaximumBatchingWindowInSeconds`
-  - `ParallelizationFactor`
-  - `ScalingConfig`
-  - `TumblingWindowInSeconds`.
+- [Self-Managed Apache Kafka](https://docs.aws.amazon.com/lambda/latest/dg/with-kafka.html)
+
+### Coverage
```

Comment:
From @joe4dev:

@HarshCasper How do we best present such a feature model to the user?

We are trying to improve the transparency regarding the supported behavioral features of Lambda Event Source Mapping beyond the simple list of supported sources we currently have in the Lambda docs [here](https://docs.localstack.cloud/user-guide/aws/lambda/#lambda-event-source-mappings). Should we just add the table below to the Lambda docs (similar than the [API coverage docs](https://docs.localstack.cloud/references/coverage/coverage_lambda/), but for behavior)